### ISSUE TITLE
GUI start with workspace argument has issues (recent files, auto-start)

### ISSUE THREAD

**Issue by Sciss** [Sciss/Mellite#126]
In GitLab by @Sciss on Aug 18, 2020, 23:47

This was probably introduced by fiddling around with `--headless`. If not using headless and specifying a workspace, we now get this:

```
Warning: auto-run object '' does not exist.
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at de.sciss.desktop.impl.RecentFilesImpl.$anonfun$updateEntries$1(RecentFilesImpl.scala:93)
	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:190)
	at de.sciss.desktop.impl.RecentFilesImpl.de$sciss$desktop$impl$RecentFilesImpl$$updateEntries(RecentFilesImpl.scala:79)
	at de.sciss.desktop.impl.RecentFilesImpl$$anonfun$1.$anonfun$applyOrElse$1(RecentFilesImpl.scala:52)
	at scala.swing.Swing$$anon$1.run(Swing.scala:37)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by Sciss** [comment]
In GitLab by @Sciss on Aug 19, 2020, 01:14

could be a problem of not giving the absolute path (parent is null)

**Comment by Sciss** [comment]
In GitLab by @Sciss on Aug 19, 2020, 01:29

mentioned in commit 9ef32943a67c2778bd2d8a636fd71a57f18904cd

**Comment by Sciss** [comment]
In GitLab by @Sciss on Aug 22, 2020, 17:18

closed via commit 9ef32943a67c2778bd2d8a636fd71a57f18904cd
